x/ethbridge/types/claim_type.go

```diff
@@ -7,11 +7,11 @@ import (
 )
 
 // ClaimType is an enum used to represent the type of claim
-type ClaimType int
+// type ClaimType int
 
 const (
-	LockText = ClaimType(iota)
-	BurnText
+	LockText = ClaimType_CLAIM_TYPE_LOCK
+	BurnText = ClaimType_CLAIM_TYPE_BURN
 )
```

Comment:
this could be removed and the enum used directly

Reply:
Discovered a great "inline" refactor tool in goland 🙌🏻